packages/create-amplify/src/amplify_project_creator.ts

```diff
@@ -30,52 +31,53 @@ export class AmplifyProjectCreator {
     private readonly npmInitializedEnsurer: NpmProjectInitializer,
     private readonly tsConfigInitializer: TsConfigInitializer,
     private readonly gitIgnoreInitializer: GitIgnoreInitializer,
-    private readonly projectRoot: string,
-    private readonly logger: typeof console = console
+    private readonly projectRoot: string
   ) {}
 
   /**
    * Executes the create-amplify workflow
    */
   create = async (): Promise<void> => {
-    this.logger.log(`Validating current state of target directory...`);
+    logger.debug(`Validating current state of target directory...`);
     await this.projectRootValidator.validate();
 
     await this.npmInitializedEnsurer.ensureInitialized();
 
-    this.logger.log(
-      `Installing packages ${this.defaultProdPackages.join(', ')}...`
-    );
+    logger.startAnimatingEllipsis(`Installing required dependencies`);
```

Comment:
The pattern for these should be
```
try {
 start
 dosomething
 } finally {
  stop
 }
```

or `await logger.withEllipsis(()=>{ do work });`

I think the latter is easier to enforce so I'd do that.

Reply:
Done in [c4e4c3f](https://github.com/aws-amplify/amplify-backend/pull/595/commits/c4e4c3ffcd020958ce25aa305f23754b39f08209)